site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -31,8 +30,14 @@
         $(function() {
             $("#submit").click(function(e){ // Submit button
                 var requestedPath = document.getElementById("input_provide_full_path").value;
-                makeSubmission(requestedPath);
-                e.stopPropagation();
+                var n = requestedPath.indexOf("..");
+                if (n != -1) {
+                    alert(".. is not supported in the path");
```

Comment:
This should really be on the php side to make it more secure (interface can't submit paths with ".." but php will accept them with the right request)